src/acl/Acl.h

```diff
@@ -26,7 +26,7 @@ namespace Acl {
 /// the ACL type name known to admins
 typedef const char *TypeName;
 /// a "factory" function for making ACL objects (of some ACL child type)
-typedef ACL *(*Maker)(TypeName typeName);
+typedef Acl::ACL *(*Maker)(TypeName typeName);
 /// use the given ACL Maker for all ACLs of the named type
```

Comment:
Please do not add unnecessary `Acl::` namespace prefixes inside the Acl namespace, like the Maker declaration line above. This change request applies to all such additions.

The current PR code relies on the fact that moving ACL inside the namespace already avoids the clash with Windows ACL typedef. We do not have to always prefix inside the namespace, but if we had to, the current PR code would not work (because it does not and _cannot_ always prefix). <!-- https://godbolt.org/z/5YnaEhMd3 -->